notes/Module-2/2.4.3-Session_based_auth.md

```diff
@@ -0,0 +1,12 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy|17/05/21|Basic Auth does not have the concept of a logout - you have to exit the browser to logout. Give REST APIS should be stateless, I am not sure this lesson is necessary. It can get quite complicated implementing sessions in Spring (https://www.javainuse.com/spring/springboot_session). I think we should just jump straight into OAuth as feedback from apprentices was they wanted more time on the Banking App and working with OAuth. See https://restfulapi.net/statelessness.  |
```

Comment:
Agree lets get them to the banking app sooner.

Reply:
Dan wanted to keep this so have left for now but will keep in coach notes this is optional and can be skipped.